nodewatcher/modules/frontend/ip_space/static/js/hilbert2d.js

```diff
@@ -0,0 +1,148 @@
+/* Author: https://github.com/ryan-williams
+   from: https://github.com/ryan-williams/hilbert-js
+   changes: This used to be a require/node module, I adapted it for in browser module, removed 3d hilbert curve, removed unneeded hilbert.js which was used for init, as I only need one module and not two.
```

Comment:
Do the same here: https://github.com/wlanslovenija/hilbert-js

Create a branch `wlanslovenija`, and add all changes there. Then use git submodule to pull that code in.